biodatasets/scifact/scifact.py

```diff
@@ -0,0 +1,428 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and
+#
+# * Nicholas Broad (nbroad1881)
```

Comment:
in the old template we were asking people to add names, but eventually decided to just use github to track contributions. can you make this consistent with the current template please https://github.com/bigscience-workshop/biomedical/blob/master/templates/template.py